pytest_monitor/pytest_monitor.py

```diff
@@ -73,14 +135,22 @@ def pytest_runtest_setup(item):
     """
     if not PYTEST_MONITORING_ENABLED:
         return
-    item_markers = {mark.name: mark for mark in item.iter_markers() if mark and mark.name.startswith('monitor_')}
+    item_markers = {
+        mark.name: mark
+        for mark in item.iter_markers()
+        if mark and mark.name.startswith("monitor_")
+    }
     mark_to_del = []
     for set_marker in item_markers.keys():
         if set_marker not in PYTEST_MONITOR_VALID_MARKERS:
-            warnings.warn("Nothing known about marker {}. Marker will be dropped.".format(set_marker))
+            warnings.warn(
+                f"Nothing known about marker {set_marker}. Marker will be dropped."
```

Comment:
**issue**: This will cause python 3.5 being unsupported. At the moment, I feel more comfortable with deprecating it for next version rather than purely and simply drop it.